### ISSUE TITLE
Bug Report: make pre-config results in cannot create directory ‘config’: File Exists

### ISSUE THREAD

**Issue by DeadSuperHero** [bonfire-networks/bonfire-app#104]
Hey all!

I'm trying to following along with [this guide](https://github.com/bonfire-networks/bonfire-app/blob/main/docs/HACKING.md) to set up a local dev installation. When I get to the `make pre-config` step, the app reports the following error:

```
Setting flavour to flavours/coordination
mkdir: cannot create directory ‘config’: File exists
make: *** [Makefile:58: pre-init] Error 1
```

I get the feeling I'm doing something wrong here, but I'm not sure about what I need to do to rectify it.

**Comment by DeadSuperHero** [comment]
It looks like this only affects `coordination` and not `classic`.

**Comment by jjl** [comment]
the pre-config step creates a symlink to the current flavour at ./config. were you using classic before switching?

**Comment by DeadSuperHero** [comment]
This was only my initial setup, but I think it might default to classic in the beginning? But yes, this problem only seems to emerge after switching to coordination, regardless of whether `mix deps.get` or even `make pre-config` is even run.

**Comment by jjl** [comment]
yes, it defaults to classic, but unless you do something that uses it, it shouldn't make a difference.

i am not sure of the current state of the coordination flavour to be honest. the team has been focusing on classic recently and there have been a bunch of changes. i'd recommend using classic for now.

you should be able to just delete the symlink at ./config and try again.

**Comment by mayel** [comment]
I am confused by the error message you're seeing, as the line of Makefile it references is `mkdir -p config/prod` which should recursively create the directories in that path, but shouldn't error if any of them already exist. Could you check if ./config is a directory or a valid symlink to a directory on your system? Also what OS/disto are you running?

**Comment by ajmeese7** [comment]
@DeadSuperHero what OS are you running this on? If it's a WSL distro then my latest PR #111 should fix this issue

**Comment by mayel** [comment]
@DeadSuperHero could you please let us know if this is now fixed?
